Blink/TermController.swift

```diff
@@ -310,20 +317,21 @@ extension TermController: TermDeviceDelegate {
    Presents a UserNotification with the `title` & `body` values passed on `data`. Tapping on the notification opens the terminal that originated the notification. Also triggered when the terminal receives a standard `OSC` sequence & iTerm2-like notification.
    
    - Parameters:
-    - data: Set the `title` and `body` String values to display those values in the notification banner.
+    - data: Set the `title` and `body` String values to display those values in the notification banner. Set the `type`'s rawValue of `BKNotificationType` to identify the type of notification used.
    */
   func viewNotify(_ data: [AnyHashable : Any]!) {
     
-    /// Don't show anything if OSC notifications have been deactivated
-    guard BKDefaults.isOscNotificationsOn() else {
+    guard let notificationTypeRaw = data["type"] as? Int, let notificationType = BKNotificationType(rawValue: notificationTypeRaw) else {
       return
     }
     
-    /**
-     - Show notification if terminal is in focus
-     - Don't show notification if terminal is not in focus and user has deactivated background notifications
-     */
-    guard _termView.isFocused() || BKDefaults.isNotificationOnBellUnfocusedOn() else {
+    /// Only show bell notifications if it's enabled & term is unfocused & type it's `BKNotificationType.bell`
+    let shouldShowBellNotifications = BKDefaults.isNotificationOnBellUnfocusedOn() && !_termView.isFocused() && notificationType  == BKNotificationType.bell
+    
+    /// Only show OSC notifications if it's enabled and if the notification type is nil (it's only used for `BEL` notifications
+    let shouldShowOscNotifications = BKDefaults.isOscNotificationsOn() && notificationType  == BKNotificationType.osc
+    
+    guard  (shouldShowBellNotifications) || (shouldShowOscNotifications) else {
```

Comment:
No need for extra braces here. And I'd rather check and return without extra variables.

smth like that:
```swift
if notificationType  == .bell && (_termView.isFocused() || !BKDefaults.isNotificationOnBellUnfocusedOn()) {
   return
}
```